components/Downloads/DownloadHeader/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { FormattedMessage, useIntl } from 'react-intl';
+import styles from './index.module.scss';
+import SectionTitle from '../../Common/SectionTitle';
+import type { NodeReleaseData } from '../../../types';
+import type { FC } from 'react';
+
+type DownloadHeaderProps = {
+  release?: NodeReleaseData;
+};
+
+const DownloadHeader: FC<DownloadHeaderProps> = ({
+  release,
+}: DownloadHeaderProps) => (
+  <>
+    <div className={styles.downloadHeader}>
+      <SectionTitle
+        path={[
+          'home',
+          useIntl().formatMessage({
```

Comment:
That's not how `useIntl` should be used. This is a Hook violation. Please read how to use React Hooks properly.